pkg/volume/flexvolume/driver-call.go

```diff
@@ -235,3 +241,29 @@ func handleCmdResponse(cmd string, output []byte) (*DriverStatus, error) {
 
 	return &status, nil
 }
+
+// getDriverCapabilities returns the reported capabilities as returned by driver's init() function
+func getDriverCapabilities(plugin *flexVolumePlugin) (*driverCapabilities, error) {
+	call := plugin.NewDriverCall(initCmd)
```

Comment:
I will admit that init() being called twice bothered me as well, however the existing isAttachable() that I removed was also instigating a double init().  There is a bit of a divide when calling down into Init() -> call.Run() as there is no easy way to assign the driver's capabilities to the flexVolumePlugin instance.  This is why I decided to handle the capabilities in NewFlexVolumePlugin() in an effort to avoid refactoring the flow.

Overall, I was trying to minimize the amount of code change to wire in this new capability.  In a future version, I can look at pushing down the flexVolumePlugin instance down into the call.Run() path, or possibly return a capabilities struct from that call.  One way or another, I'd have to bridge that divide.